commands/_copilot_modules/utils.py

```
@@ -0,0 +1,300 @@
+utils.py - Shared utilities for modular copilot commands
+
+Provides common functionality for:
+- GitHub API operations
+- Git command wrappers
+- JSON schema validation
+- Common error messages
+"""
+
+import json
+import subprocess
+import sys
```

Ruff (0.12.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)